### ISSUE TITLE
Consistent Indentification example 8 is a Label In Name failure

### ISSUE THREAD

**Issue by fstrr** [w3c/wcag#4250]
[Example 8: Failure primarily impacting assistive technology users](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html#dfn-set-of-web-pages:~:text=Example%208%3A%20Failure%20primarily%20impacting%20assistive%20technology%20users)

> "Two buttons with the same functionality visually have the same text, but have been given different aria-label="..." accessible names. For users of assistive technologies, these two buttons will be announced differently and inconsistently."

We could remove the example or change the description to say that this is also a failure of Label In Name.

**Comment by fstrr** [comment]
[Example 2: Check Mark](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html#dfn-same-functionality:~:text=Example%202%3A%20Check%20Mark)

> A check mark icon functions as "approved", on one page but as "included" on another. Since they serve different functions, they have different text alternatives.

Maybe I'm missing something (I probably am), but I'm not 100% sure if this relevant as it's just images rather than images inside a component. Every other example involves interactive components (links, buttons, form controls, etc.).

**Comment by fstrr** [comment]
@patrickhlauke If you do update this, while you're in there what about:

- updating the "A Web mail program built using Asynchronous JavaScript and XML (AJAX)" text so it doesn't sound so "woo, web 2.0 is amazing"?
- Changing the http to https in a couple of examples
- fixing the casing of Web and Success Criterion in a few places

**Comment by patrickhlauke** [comment]
> @patrickhlauke If you do update this, while you're in there what about:
> 
> updating the "A Web mail program built using Asynchronous JavaScript and XML (AJAX)" text so it doesn't sound so "woo, web 2.0 is amazing"?
> Changing the http to https in a couple of examples
> fixing the casing of Web and Success Criterion in a few places

I'm assuming this was already taken care of at some point, as i can't find what you're referring to in the 3.2.4 understanding

EDIT: ah, those are in the definition for "web page" ... so would need to be tackled separately as an erratum (which no doubt will cause a whole "oh why do we even bother to do this?" pushback from...folks)

**Comment by patrickhlauke** [comment]
anyway, PR here https://github.com/w3c/wcag/pull/5022
